plugins/vli/vli-cablematters.quirk

```diff
@@ -0,0 +1,4 @@
+# Cable Matters
+[USB\VID_2BD5&PID_5240]
+Plugin = vli
+GType = FuVliPDDevice
```

Comment:
Woah, hang on a minute -- isn't this suppose to be FuVliPdDevice rather than FuVliPDDevice? I see the same typo in plugins/vli/vli-fujitsu.quirk

Reply:
Thank you for the catch!

Reply:
Maybe we want some check-source.py check for typos like this in the quirks?

Reply:
https://github.com/fwupd/fwupd/pull/7817